### DISCUSSION TITLE
blog/running-pace-calculator/

### DISCUSSION THREAD

**Discussion by giscus** [AntoineSoetewey/statsandr#88] [General]
# blog/running-pace-calculator/

Type in the distance and the time you plan to run and this running pace calculator will give you your necessary pace and the splits

https://statsandr.com/blog/running-pace-calculator/

**Comment by AntoineSoetewey** [comment]
Relevant and respectful comments are welcome. Off-topic comments or spam will be deleted. Thank you for your understanding!

**Reply by jugheadjones10** [reply]
Woah

**Reply by jugheadjones10** [reply]
Nice
